package.json

```diff
@@ -71,17 +75,20 @@
     "js-yaml": "^3.13.1",
     "json-stable-stringify-without-jsonify": "^1.0.1",
     "levn": "^0.4.1",
-    "lodash": "^4.17.21",
+    "map-values": "^1.0.1",
     "minimatch": "^3.0.4",
     "natural-compare": "^1.4.0",
+    "omit": "^1.0.1",
     "optionator": "^0.9.1",
     "progress": "^2.0.0",
     "regexpp": "^3.1.0",
+    "rfdc": "^1.3.0",
```

Comment:
I think `deepExtend({}, obj)` is equivalent to `clone(obj)`. Can this dependency be dropped too?

Reply:
Resolved in f9ac034a2.